module/zcommon/Makefile.in

```diff
@@ -15,3 +15,5 @@ $(MODULE)-objs += zfs_comutil.o
 $(MODULE)-objs += zfs_fletcher.o
 $(MODULE)-objs += zfs_uio.o
 $(MODULE)-objs += zpool_prop.o
+
+$(MODULE)-$(CONFIG_X86) += zfs_fletcher_intel.o
```

Comment:
@jxiong are to open to renaming this `zfs_fletcher_avx2`, or would you prefer to leave it as is?